### PR TITLE
Fix electron deprecations

### PR DESCRIPTION
**Pull request by mitchhentges**
Allegedly, [if you were on `v0.37.8`, you'd be notified of deprecated API usage](https://github.com/electron/electron/releases/tag/v1.0.0).
Welp, we didn't have any errors, but updating to electron `v1.0.0` wasn't a safe move. This fixes the broken bits

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import remote from 'remote';
-const app = remote.require('app');
+const {app} = require('electron').remote;
```
Comment (tomitm) [inline src/constants.js]: Oh cool, we no longer need to do `remote.require`? Can you split this into the following:

```js
import { remote } from 'electron';
const { app } = remote;
```

Just want to standardize on `import` syntax over `require` if we no longer need it.

Comment (tomitm) [inline src/constants.js]: Also, I think I'm inconsistent with spacing within `{...}`. Anyone have a preference? Pretty sure we can enforce it with eslint.

**Comment by tomitm** [comment]
Ready to merge this in once you've fixed that one comment. :+1:

**Comment by mitchhentges** [comment]
Fixed!

File: src/keyboard.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-const ipc = require('electron').ipcRenderer;
+import { ipcRenderer } from 'electron';
```
Comment (tomitm) [inline src/keyboard.js]: Don't need to change this, but just so you know, you could've used `import { ipcRenderer as ipc } from 'electron'` to keep the name!

**Pull request merged**